src/app/beer_garden/scheduler.py

```diff
@@ -530,6 +556,48 @@ def create_job(job: Job) -> Job:
     return job
 
 
+def create_jobs(jobs: JobDefinitionList) -> JobIDList:
```

Comment:
Probably obvious by this point, but this should just take a list of `Job` objects and return a list of string ids.

Reply:
Done.